python/triton/language/core.py

```diff
@@ -31,7 +31,7 @@ def _to_tensor(x, builder):
         return _to_tensor(x.value, builder)
     elif isinstance(x, tensor):
         return x
-    assert False, f'cannot convert {x} to tensor'
+    assert False, f"cannot convert {x} of type {type(x)} to tensor"
```

Comment:
When mixing some triton-unsupported-type values to the JIT function's arguments (e.g. a `numpy.int64` integer 32), the `_to_tensor` will fail with some error information like "cannot convert 32 to tensor", this is a bit confusing. By adding the type to the information will make it more intuitive I think.